### DISCUSSION TITLE
Disable a selection option?

### DISCUSSION THREAD

**Discussion by carter2dev** [themesberg/flowbite-svelte#1449] [Q&A]
I might be crazy here. But I cannot find any way nor documentation how I can disable options in a Select component?

**Comment by shinokada** [comment]
Will `disabled` work for you?

```
  <Select disabled class="mt-2"  />
```

**Comment by carter2dev** [comment]
hi @shinokada no. that will disable the entire select component. I need to disable specific options based on conditions
